docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -22,17 +22,24 @@ When your project requires custom native code, a config plugin, a custom runtime
 
 A **development build** of your app is a Debug build that contains the `expo-dev-client` package. As a production build is for the general public, and a preview build lets your team test your next release, a development build lets developers iterate as quickly as possible. It comes with extensible development tools to develop and test your project.
 
-You can think of a development build as your version of the Expo Go client.
+You can think of a development build as your fully customizable version of Expo Go.
 
 ## What is expo-dev-client
 
-The [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) package is used to create a development build. It is a library designed to support any workflow, release process, or set of dependencies in the Expo/React Native ecosystem. Whatever your project needs, either now or in the future, you'll be able to create a development build for it and get the productivity and quality of life improvements of JavaScript-driven development.
+The [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) package replaces the default in-app development tools UI that React Native provides (the ["Dev Menu"](https://reactnative.dev/docs/debugging#accessing-the-dev-menu)) with [a more powerful and extensible UI](/debugging/tools/#developer-menu) and adds a launcher UI, so you can launch updates (such as from [PR previews](/develop/development-builds/development-workflows/#pr-previews)) and switch between development servers without needing to rebuild the app binary.
```

Comment:
```suggestion
The [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) package replaces the default in-app development tools UI that React Native provides (the [Dev Menu](https://reactnative.dev/docs/debugging#accessing-the-dev-menu)) with [a more powerful and extensible UI](/debugging/tools/#developer-menu) and adds a launcher UI, so you can launch updates (such as from [PR previews](/develop/development-builds/development-workflows/#pr-previews)) and switch between development servers without needing to rebuild the app binary.
```

Reply:
@brentvatne, I am confused by "switch between development servers  without needing to rebuild the app binary". If I am creating a development build, I'd expect to only run one development server to run my project in the development mode? Or maybe I am missing the context here.

Reply:
- when you build an app without expo-dev-client, ie: using built-in react-native dev tools ui, it points to a single dev server host location -- wherever it existed when you started your server. so if you're running it over LAN for example, and your LAN ip changes, you need to rebuild
- when using expo-dev-client, you can close / open dev server, switch ip / switch to tunnel / load an update / load someone elses dev server remotely

Reply:
> so if you're running it over LAN for example, and your LAN ip changes, you need to rebuild

TIL! Thank you for providing context!